app/src/main/java/com/developersbreach/composeactors/ui/screens/favorites/tabs/FavoriteMoviesTabContent.kt

```diff
@@ -25,36 +32,61 @@ import com.developersbreach.composeactors.ui.theme.ComposeActorsTheme
  * Content shown for home tab Favorites.
  * If user did not add any movies to favorites, a message will be shown.
  */
+@OptIn(ExperimentalFoundationApi::class)
 @Composable
 fun FavoriteMoviesTabContent(
     getSelectedMovieDetails: (Int) -> Unit,
     favoriteMovies: List<Movie>,
+    removeFavoriteMovie: (Movie) -> Unit,
 ) {
     if (favoriteMovies.isEmpty()) {
         NoFavoritesFoundUI()
     }
 
-    LazyVerticalGrid(
-        columns = GridCells.Fixed(3),
-        modifier = Modifier.fillMaxWidth(),
-        horizontalArrangement = Arrangement.spacedBy(12.dp),
-        verticalArrangement = Arrangement.spacedBy(12.dp),
-        contentPadding = PaddingValues(vertical = 2.dp, horizontal = 16.dp)
+    LazyColumn( modifier = Modifier
+        .fillMaxWidth()
     ) {
-        items(
-            items = favoriteMovies,
-            key = { it.movieId }
-        ) { movieItem ->
-            LoadNetworkImage(
-                imageUrl = movieItem.posterPathUrl,
-                contentDescription = stringResource(R.string.cd_movie_poster),
-                shape = MaterialTheme.shapes.medium,
+        items(favoriteMovies.size) { index ->
+            val movieItem = favoriteMovies[index]
+            Box(
                 modifier = Modifier
-                    .size(120.dp, 180.dp)
-                    .clickable {
-                        getSelectedMovieDetails(movieItem.movieId)
-                    }
-            )
+                    .fillMaxWidth()
+                    .height(200.dp)
+                    .animateItemPlacement(),
+                contentAlignment = Alignment.BottomStart
+            ) {
+                LoadNetworkImage(
+                    imageUrl = movieItem.posterPathUrl,
```

Comment:
As we've discussed, we have to use the movie banner instead of poster here.
The object `Movie` right now only have 
```
val posterPathUrl: String
```

But we actually need banner property first in the Movie object.
Take reference of `MovieDetail` object, there we have the property
```
val banner: String
```
So from add this property in `Movie` object and  make use of banner.